indico/modules/search/client/js/components/results/Event.jsx

```diff
@@ -76,15 +79,14 @@ const personShape = PropTypes.shape({
 
 Event.propTypes = {
   title: PropTypes.string.isRequired,
-  url: PropTypes.string.isRequired,
-  typeFormat: PropTypes.oneOf(['lecture', 'meeting', 'conference']).isRequired,
+  eventId: PropTypes.number.isRequired,
+  typeFormat: PropTypes.oneOf(['lecture', 'meeting', 'conference']),
```

Comment:
when is this missing?

and just wondering, why `typeFormat` and not `type`?

Reply:
Not supposed to be missing - however, the component should work completely fine without it, for that reason I decided to default it to minimal behaviour.

Reply:
Not supposed to be missing, however since the component works just fine without it, I decided to default it to a minimal behaviour.

Reply:
Forgot the second line. `type` is supposed to be a generic type that identifies the record, such as, Event, Contribution, Attachment. `typeFormat` is a record-specific format such as Event being a meeting, conference, etc.

Reply:
ah... unless citadel uses `type_format` by default, maybe `event_type` (`eventType` here) would be clearer?

Reply:
Yes, citadel is already using it. The problem is `type_format` is already being indexed by all records, and it applies to both a contribution, event and attachment independently of their type, so changing the prop wouldn't be simple. This would have to be a change at schema-citadel level.

Reply:
let's leave it then; i was just curious about the naming :)